scripts/cloud-e2e-beta.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# set exit on error to true
+set -e
```

Comment:
This and the other e2e file are extremely similar. Can we extract these to a shared bash function and parameterize the values that are different?